freedomofpress/securedrop-client
# ensure safe perms for svs.sqlite and sync_flag

# Description

Fixes https://github.com/freedomofpress/securedrop-client/issues/1232

# Test Plan

- Build deb for the client and install on sd-app or just patch it with this PR change
- Delete ~/.securedrop_client (you can manually copy config.json if you want but it's not necessary to test this change)
- Start the client via commandline by running: `securedrop-client`
- [ ] Confirm svs.sqlite and sync_flag now have 600 perms instead of 644
